project/Dependencies.scala

```diff
@@ -21,41 +21,52 @@ object Dependencies {
     val kinesisClient    = "1.9.1"
     val kinesisConnector = "1.3.0"
     val validator        = "2.2.6"
-    val elasticsearch    = "6.2.4"
+    val elasticsearch    = "6.3.2"
     val nsqClient        = "1.1.0-rc1"
+    val jackson          = "2.9.6"
+
+    // TODO: consider removing paranamer version override when upgrading other libaries
+    val paranamer = "2.8" // Jackson 2.9.6 uses version 2.8 of paranamer internally
+                          // but there is json4s-core_2.12 3.2.11 that overrides it back to 2.6
+
     // Scala
     val scopt            = "3.7.0"
     val scalaz7          = "7.2.22"
     val snowplowTracker  = "0.5.0"
     val analyticsSDK     = "0.3.0"
     val awsSigner        = "0.5.0"
-    val elastic4s        = "6.2.4"
+    val elastic4s        = "6.3.6"
     val pureconfig       = "0.9.1"
     // Scala (test only)
     val specs2           = "4.1.0"
   }
 
   object Libraries {
     // Java
-    val config           = "com.typesafe"            %  "config"                       % V.config
-    val slf4j            = "org.slf4j"               %  "slf4j-simple"                 % V.slf4j
-    val log4jOverSlf4j   = "org.slf4j"               %  "log4j-over-slf4j"             % V.slf4j
-    val kinesisClient    = "com.amazonaws"           %  "amazon-kinesis-client"        % V.kinesisClient
-    val kinesisConnector = "com.amazonaws"           %  "amazon-kinesis-connectors"    % V.kinesisConnector
-    val validator        = "com.github.fge"          %  "json-schema-validator"        % V.validator
-    val elasticsearch    = "org.elasticsearch"       %  "elasticsearch"                % V.elasticsearch
-    val nsqClient        = "com.snowplowanalytics"   %  "nsq-java-client_2.10"         % V.nsqClient
+    val config           = "com.typesafe"                     %  "config"                       % V.config
+    val slf4j            = "org.slf4j"                        %  "slf4j-simple"                 % V.slf4j
+    val log4jOverSlf4j   = "org.slf4j"                        %  "log4j-over-slf4j"             % V.slf4j
+    val kinesisClient    = "com.amazonaws"                    %  "amazon-kinesis-client"        % V.kinesisClient
+    val kinesisConnector = "com.amazonaws"                    %  "amazon-kinesis-connectors"    % V.kinesisConnector
+    val validator        = "com.github.fge"                   %  "json-schema-validator"        % V.validator
+    val elasticsearch    = "org.elasticsearch"                %  "elasticsearch"                % V.elasticsearch
+    val nsqClient        = "com.snowplowanalytics"            %  "nsq-java-client_2.10"         % V.nsqClient
+    val paranamer        = "com.thoughtworks.paranamer"       %  "paranamer"                    % V.paranamer
+    val jacksonCbor      = "com.fasterxml.jackson.dataformat" %  "jackson-dataformat-cbor"      % V.jackson
```

Comment:
due to `com.fasterxml.jackson.dataformat` length I had to add space to keep the structure.